packages/spatial/src/renderer/components/PostProcessingComponent.tsx

```diff
@@ -63,10 +83,220 @@ export const PostProcessingComponent = defineComponent({
 
     useEffect(() => {
       if (!rendererEntity) return
-      configureEffectComposer(
-        rendererEntity,
-        postprocessingComponent.enabled.value ? postprocessingComponent.effects.get(NO_PROXY_STEALTH) : undefined
-      )
+      //configureEffectComposer(
```

Comment:
we should definitely keep this as a separate function, long useEffects are hard to read